packages/combobox/src/Combobox/Combobox.styles.ts

```diff
@@ -16,25 +18,54 @@ import { ComboboxSize as Size, Overflow } from '../Combobox.types';
 // Rename the variable defined in chip styles
 const inputHeight = chipHeight;
 
+const flexGap = 4;
+
 /**
  * Width of the widest character (in px)
  */
 export const maxCharWidth: Record<Size, number> = {
+  [Size.XSmall]: typeScales.body1.fontSize,
+  [Size.Small]: typeScales.body1.fontSize,
   [Size.Default]: typeScales.body1.fontSize,
   [Size.Large]: typeScales.body2.fontSize,
 };
 
 /**
  * Size of combobox x & y padding (in px)
+ * (wrapperHeight - inputHeight - (borderTop + borderBottom)) / 2
  */
-export const comboboxPadding: Record<Size, { x: number; y: number }> = {
+export const comboboxPadding: Record<
+  Size,
+  {
+    y: number;
+    xLeftWithChip: number;
+    xLeftWithoutChip: number;
+    xRight: number;
+  }
+> = {
+  [Size.XSmall]: {
+    y: (22 - inputHeight[Size.XSmall] - 2) / 2, // (22 - 18 - 2 ) / 2 = (2) / 2 = 1
+    xLeftWithChip: 1,
+    xLeftWithoutChip: 10,
+    xRight: 4,
+  },
+  [Size.Small]: {
+    y: (28 - inputHeight[Size.Small] - 2) / 2, // (28 - 20 - 2) / 2 = (6) / 2 = 3
+    xLeftWithChip: 4,
+    xLeftWithoutChip: 10,
+    xRight: 8,
+  },
   [Size.Default]: {
-    y: (36 - inputHeight[Size.Default] - 2) / 2,
-    x: spacing[2] - 1,
+    y: (36 - inputHeight[Size.Default] - 2) / 2, // (36 - 24 - 2) / 2 = (10) / 2 = 5
```

Comment:
The logic of `get padding from totalHeight - otherStuff` and the other direction of `get height from contentHeight + padding` feels reused throughout. Maybe it'd be more legible to use a util fn? These manual math comments make me 😵‍💫😵‍💫